devimint/src/vars.rs

```diff
@@ -120,7 +120,8 @@ declare_vars! {
 
         // fedimint config variables
         FM_TEST_BITCOIND_RPC: String = "http://bitcoin:bitcoin@127.0.0.1:18443";
```

Comment:
I saw this replaced elsewhere, is there a reason it's still around?

Reply:
Yes, in some of our integration tests we have a different RPC interface that communicates directly with bitcoin RPC to perform actions such as mining new blocks.  Since we only implemented that interface for bitcoind we need to pass in the bitcoind URL for tests only.